dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java

```diff
@@ -104,32 +133,35 @@ public void testElasticSearchHealthCheck() {
             assertTrue("Should fail", true);
         }
 
+        // Register and unpublish workflow
+        workflowApi.manualRegister("github", "DockstoreTestUser2/dockstore_workflow_cnv", "/workflow/cnv.cwl", "", "cwl", "/test.json");
+        final Workflow workflowByPathGithub = workflowApi.getWorkflowByPath(WorkflowIT.DOCKSTORE_TEST_USER2_RELATIVE_IMPORTS_WORKFLOW, null);
+        // do targetted refresh, should promote workflow to fully-fleshed out workflow
+        final Workflow workflow = workflowApi.refresh(workflowByPathGithub.getId());
+        workflowApi.publish(workflow.getId(), new PublishRequest() {
+            public Boolean isPublish() { return false;}
+        });
+
         // Update the search index
         extendedGa4GhApi.toolsIndexGet();
+        // Should it though? Shouldn't this be healthy, just empty?
```

Comment:
I think the elastic search health check was supposed to fail when elastic search was completely empty (i.e. we forget to run the initial index after deploying).

That said, now the order is flipped